### PR TITLE
Create MGHPCCOrigin.yaml and fix cache

### PR DESCRIPTION
**Pull request by biozit**

### CODE REVIEW

File: virtual-organizations/IceCube.yaml [hunk 1]
```diff
@@ -58,13 +58,24 @@ DataFederations:
         AllowedOrigins:
           - IceCube_StashCache_origin
           
-      - Path: /icecube/NRP-UNL/PROTECTED
+      - Path: /nrpnb/protected/icecube/
```
Comment (bbockelm) [inline virtual-organizations/IceCube.yaml:61]: The custom is to have the top-level component of the origin reflect the VO.  It appears this is indicating the origin.

Can you do:

```
/icecube/nrpnb/protected
```

instead?

Note XRootD has various mechanisms to add or subtract prefixes before it tries to open the file on disk.

File: virtual-organizations/IceCube.yaml [hunk 2]
```diff
@@ -58,13 +58,24 @@ DataFederations:
...
         AllowedOrigins:
           - NEBRASKA_NRP_OSDF_ORIGIN
         AllowedCaches:
           - ANY
+          
+      - Path: /mghpcc/protected/icecube/
+        Authorizations:
+          - SciTokens:
+              Issuer: https://chtc.cs.wisc.edu/icecube
+              Base Path: /mghpcc/protected/icecube/
```
Comment (bbockelm) [inline virtual-organizations/IceCube.yaml:76]: Similar to above -- make the base component of the namespace the VO not the origin name.

**Review by bbockelm — CHANGES_REQUESTED** [review]
I'd like to get at least the top-level of the namespace to align with the best practices on the OSDF.

File: virtual-organizations/IceCube.yaml (hunk 1 above)
Comment (biozit) [reply virtual-organizations/IceCube.yaml]: sure, solved!

File: virtual-organizations/IceCube.yaml (hunk 2 above)
Comment (biozit) [reply virtual-organizations/IceCube.yaml]: sure, solved!

**Comment by biozit** [comment]
Please, is there anything to fix here? @bbockelm @djw8605

**Review by bbockelm — APPROVED** [review]
LGTM.  Ready to merge.

**Pull request merged**